src/notation/internal/notationactioncontroller.cpp

```diff
@@ -964,6 +964,45 @@ void NotationActionController::move(MoveDirection direction, bool quickly)
             return;
         }
 
+        if (playbackController()->isPlaying()) {
+            // Use current playback location (instead of selection) to move
+            auto beat = m_playbackController->currentBeat();
```

Comment:
It's preferable to use `playbackController()` instead of `m_playbackController`. Actually, `m_playbackController` is considered an implementation detail.